kefir.js.flow

```diff
@@ -104,6 +104,7 @@ declare class Observable<+V,+E=*> {
   slidingWindow(max: number, min?: number): Observable<V[],E>;
   bufferWhile(predicate?: (value: V) => boolean, options?: {flushOnEnd?: boolean}): Observable<V[],E>;
   transduce(transducer: any): Observable<any,E>;
+  thru<V2>(callback: (Observable<V,E> => V2)): V2;
```

Comment:
Can it be written as `thru<V2>(callback: this): V2;`?

Reply:
Oh, probably. Good call.